lib/util/index.js

```diff
@@ -3,6 +3,17 @@
 let path = require("path");
 let crypto = require("crypto");
 
+exports.requireOptional = (pkg, errorMessage, supplier = pkg) => {
+	try {
+		return require(pkg);
+	} catch(err) {
+		if(err.code !== "MODULE_NOT_FOUND") {
```

Comment:
this isn't present in faucet-js implementation, but I _think_ that's more of an oversight than intentional and might lead to misleading error messages

Reply:
What is the difference to the current implementation?

Reply:
See link above: faucet-js currently doesn't distinguish, thus always swallowing exceptions here. So if you happen to have a syntax error in the target file during development, all you get is the "please install" prompt. I can't think of any other scenarios where this might matter.

Reply:
Ok, I think this new way makes sense.